### DISCUSSION TITLE
Convert from 6-dot braille coding to human-readable text?

### DISCUSSION THREAD

**Discussion by xbing6** [parkrrrr/bdio#16] [Q&A]
Hi, if I want to convert from 6-dot braille coding to human-readable text, besides google, do you have any suggestions? Thanks.

**Comment by parkrrrr** [comment]
You'll need to know more than just that it's Braille. Even within English, there are multiple mappings, and of course other languages have their own mappings. But, if you know what kind of Braille it is, I believe that [liblouis](http://liblouis.org/) does a pretty good job of translating in reverse.
